deploy/sre-pruning/115-reinstall-rmo.CronJob.yaml

```
@@ -0,0 +1,107 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: sre-rmo-cleanup-sa
+  namespace: openshift-route-monitor-operator
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: sre-rmo-cleanup-role
+  namespace: openshift-route-monitor-operator
+rules:
+- apiGroups:
+  - "operators.coreos.com"
+  resources:
+  - clusterserviceversions
+  - subscriptions
+  - installplans
+  verbs:
+  - list
+  - get
+  - delete
+  - create
+- apiGroups:
+  - "batch"
+  resources:
+  - cronjobs
+  verbs:
+  - list
+  - get
+  - delete
+- apiGroups:
+  - ""
+  resources:
+  - serviceaccounts
+  verbs:
+  - list
+  - get
+  - delete
+- apiGroups:
+  - "rbac.authorization.k8s.io"
+  resources:
+  - roles
+  - rolebindings
+  verbs:
+  - list
+  - get
+  - delete
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: sre-rmo-cleanup-rb
+roleRef:
+  kind: Role
+  name: sre-rmo-cleanup-role
+  apiGroup: rbac.authorization.k8s.io
+  namespace: openshift-route-monitor-operator
+subjects:
+- kind: ServiceAccount
+  name: sre-rmo-cleanup-sa
+  namespace: openshift-route-monitor-operator
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: sre-rmo-cleanup
+  namespace: openshift-route-monitor-operator
+spec:
+  ttlSecondsAfterFinished: 100
+  failedJobsHistoryLimit: 1
+  successfulJobsHistoryLimit: 3
+  concurrencyPolicy: Replace
+  schedule: "*/30 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          serviceAccountName: sre-rmo-cleanup-sa
+          restartPolicy: Never
+          containers:
+          - name: rmo-reinstaller
+            image: image-registry.openshift-image-registry.svc:5000/openshift/cli:latest
+            imagePullPolicy: Always
+            command:
+            - sh
+            - -c
+            - |
+              #!/bin/bash
+
+              set -euxo pipefail
+
+              COUNT_CSV=$(oc get csv | grep route-monitor-operator | wc -l) || true
```

on a different solution, you can all the namespace here
```suggestion
              COUNT_CSV=$(oc -n openshift-route-monitor-operator get csv  | grep route-monitor-operator | wc -l) || true
```